backend/.env.dist

```diff
@@ -63,7 +63,6 @@ WEBHOOK_ELOPAGE_SECRET=secret
 
 # Federation
 FEDERATION_VALIDATE_COMMUNITY_TIMER=60000
-FEDERATION_XCOM_SENDCOINS_ENABLED=false
```

Comment:
The config declaration lives now in core, but the actually .env file where to put FEDERATION_XCOM_SENDCOINS_ENABLED are still the .env from backend and maybe also frontend to show only other communities if federation xcom sendcoins is enabled. Core cannot have there own .env. On starting a node instance, it can read the .env in the working directory of the instance (backend, federation, dht-node), but core, shared and database are more like libraries.

Reply:
Ah yeah, reinserted the property again here